spec/index.bs

```diff
@@ -7960,6 +7960,27 @@ dictionary GPURenderPassColorAttachment {
         render pass. If not [=map/exist|provided=] defaults to `{r: 0, g: 0, b: 0, a: 0}`. Ignored
         if {{GPURenderPassColorAttachment/loadOp}} is not {{GPULoadOp/"clear"}}.
 
+        <div class=note>
```

Comment:
Putting everything in a note means it is non-normative (just explanation that's not technically part of the standard). Did you mean to only put the first paragraph as a note?

Reply:
Oh I'd like to make these a part of SPEC and I have moved them out of "note".